model_analyzer/analyzer.py

```diff
@@ -165,6 +173,16 @@ def analyze(self, mode, quiet):
         if not quiet:
             self._result_manager.write_results()
 
+        top_3_model_config_names = [
+            x.model_config().get_config()['name']
+            for x in self._result_manager.top_n_results(n=3)
+        ]
+        logger.info(f"Run `model-analyzer report --report-model-configs "
```

Comment:
example:

```
...
2021-11-19 01:31:40.95 INFO[analyzer.py:180] Run `model-analyzer report --report-model-configs add_sub_i0,add_sub_i1 -e /opt/triton-model-analyzer` to generate detailed reports for the 2 best configurations.
$ 
```

Reply:
Format here is different. For analyze you had  "<explaining words>, <command>".  Here you have "<command>, <explaining words>".  Lets make them consistent.  I think it should be "<explaining words>: <command>"

Reply:
reworded 👍